orion/pkg/repos/migrations/000005_create_table_users.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE users
+(
+    id          int unsigned NOT NULL AUTO_INCREMENT,
+    created_at  datetime     NOT NULL,
+    updated_at  datetime     NOT NULL,
+    deleted_at  datetime,
+    first_name  varchar(32)  NOT NULL,
+    last_name   varchar(32)  NOT NULL,
+    middle_name varchar(32),
+    email       varchar(64)  NOT NULL,
+    phone       varchar(24)  NOT NULL,
+    isGuardian  boolean      NOT NULL,
```

Comment:
We can give this a `NOT NULL DEFAULT false` constraint.

Reply:
Sorry in MySQL, it is actually `NOT NULL DEFAULT 0`. 0 means false while 1 means true.